collector/multiplier.go

```diff
@@ -0,0 +1,72 @@
+package collector
```

Comment:
Codewise LGTM, but shouldn't this rather be put in the `exporterkit` repo?

Reply:
Uh, I didn't have this on my radar. When @giantswarm/sig-monitoring wants it to be there I will port it.

Reply:
My point `prometheus` is loosely connected to `kubernetes`. I wouldn't like `operatorkit` to become `microkit2`. Having said that I'm aware `prometheus` is already a dependency of this project and I'm not clear on `exporeterkit` vision.